docs/application/dotnet/guides/app-management/component-port.md

```diff
@@ -0,0 +1,469 @@
+                _thread.Start();
+                return true;
+            }
+
+            public override void OnStartCommand(AppControl appControl, bool restarted)
+            {
+                Log.Info("OnStartCommand()");
+                base.OnStartCommand(appControl, restarted);
+            }
+
+            public override void OnDestroy()
+            {
+                Log.Info("OnDestroy()");
+                base.OnDestroy();
+
+                if (_port != null)
+                {
+                    _port.RequestReceived -= OnRequestReceived;
+                    _port.Dispose();
+                    _port = null;
+                }
+            }
+        }
+    }
+    ```
+
+2.  Set up the receiving thread of the component.
+
+    To have the receiving thread of the component wait for incoming requests, call the `WaitForEvent()` method of the `Tizen.Applications.ComponentBased.ComponentPort` class. If the `WaitForEvent()` method is called in the main thread, the next line of calling the `WaitForEvent()` method will not be called until the `Cancel()` is called. The `WaitForEvent()` method starts the main loop to wait for events from other components. To avoid blocking the main thread, it's recommended to use the `Tizen.Applications.ComponentBased.ComponentTask` class.
+    To handle access control, call the `AddPrivilege()` method of the `Tizen.Applications.ComponentBased.ComponentPort` class. If a client doesn't have a permission, the request will be rejected with the permission denied error.
+    To handle the received request, define and the register an event handler for the `RequestReceived` event of the `Tizen.Applications.ComponentBased.ComponentPort` class.
+
+    ```
+    {
+        _port = new ComponentPort("ServerService");
+        _port.RequestReceived += OnRequestReceived;
+        _port.AddPrivilege("http://tizen.org/privilege/datasharing");
+        _thread = new Thread(new ThreadStart(OnThread));
+        _thread.Start();
+    }
+
+    private void OnThread()
+    {
+        Log.Info("OnTrhead()");
+        _port.WaitForEvent();
+    }
+
+    private void OnRequestedReceived(object sender, RequestEventArgs args)
+    {
+        Log.Info("Sender: + args.Sender);
+        var request = args.Request;
+        if (request.GetType() == typeof(Request))
+        {
+            var req = (Request)request;
+            Log.Info("Command: " + req.Command);
+            Log.Info("Sequence: " + req.Sequence);
+            Log.Info("Message: " + req.Message);
+
+            if (args.IsReplyRequested)
+            {
+                args.Reply = new Response(req.Command, req.Sequence, 0);
+            }
+        }
+    }
+    ```
+
+3.  In the sending component, to send the request, follow the steps:
+
+    a. Call the `WaitForEvent()` method `Tizen.Applications.ComponentBased.ComponentPort` class in the thread if you want to receive the reply from the ServerService.
```

Comment:
```suggestion
    1. Call `WaitForEvent()` of the `Tizen.Applications.ComponentBased.ComponentPort` class in the thread, if you want to receive the reply from the **ServerService.**
```
Is ServiceService correct, I guess not, please verify.

Reply:
Done